client/layout/sidebar/footer.jsx

```diff
@@ -15,8 +15,11 @@ import HappychatButton from 'components/happychat/button';
 const SidebarFooter = ( { translate, children } ) => (
 	<div className="sidebar__footer">
 		{ children }
-		<Button compact borderless href="/help">
-			<Gridicon icon="help-outline" /> { translate( 'Help' ) }
+		<Button borderless href={ config( 'signup_url' ) + '?ref=calypso-selector' }>
```

Comment:
Oh, I think I didn't explain myself correctly — the sidebar footer area is setup in a way that any section can provide their custom functionality, while we keep some thing consistent throughout (help and chat buttons).

My idea was that when you are IN the editor we could show an add new post button, not in my sites. Starting a new post in the current site is not as straightforward as it should; the masterbar new post item forces you to reselect a site. Since "Add new site" makes sense for a footer action in `My Sites`, I was saying that "Add new post" may make sense as the primary footer action of the `Editor`.